jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -40,8 +40,11 @@
      (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args"))
    (let ((use-torso t))
      (if (and (member :use-torso args) (null (cadr (member :use-torso args)))) (setq use-torso nil))
-     (send self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t)
+     (send* self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t args)
    ))
+  (:rarm-angle-vector 
+   (av &optional (tm 3000) &rest args)
+   (send* self :angle-vector av tm :ctype :arm-controller args))
```

Comment:
How about setting `&optional (ctype nil)` args in `:angle-vector` as `robot-interface.l` and https://github.com/jsk-ros-pkg/jsk_robot/blob/master/jsk_baxter_robot/baxtereus/baxter-interface.l#L237?
When `ctype` is `nil`, `:angle-vector` uses `:default-controller`.

BTW, lisp function `equal` and `eq` is different.
If you only compare symbol, `eq` is fine, too.